pyproject.toml

```diff
@@ -8,3 +8,22 @@ exclude = '''
   | \.mypy_cache
 )/
 '''
+
+[tool.pylint.master]
```

Comment:
not sure we should enforce pylint and pyright either (redundant with flake8 and pylint)